packages/orm/src/Aliases.ts

```diff
@@ -39,28 +39,28 @@ export type Alias<T extends Entity> = {
 };
 
 export interface PrimitiveAlias<V, N extends null | never> {
-  eq(value: V | N | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  ne(value: V | N | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  in(values: V[] | undefined): ColumnCondition;
-  gt(value: V | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  gte(value: V | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  lt(value: V | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  lte(value: V | undefined | PrimitiveAlias<V, any>): ColumnCondition | RawCondition;
-  like(value: V | undefined): ColumnCondition;
-  ilike(value: V | undefined): ColumnCondition;
-  search(value: V | undefined): ColumnCondition;
-  between(v1: V | undefined, v2: V | undefined): ColumnCondition;
+  eq(value: V | N | undefined | PrimitiveAlias<V, any>): ExpressionCondition;
+  ne(value: V | N | undefined | PrimitiveAlias<V, any>): ExpressionCondition;
+  in(values: (V | null)[] | undefined): ExpressionCondition;
```

Comment:
Changing this to return an `ExpressionCondition` is what lets `in` conditionally return a `{ or: ... }` when it notices the `null` value...